modules/appeals_api/app/workers/appeals_api/add_icn_updater.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module AppealsApi
+  class AddIcnUpdater
+    include Sidekiq::Worker
```

Comment:
So by default a sidekiq worker will run for ~21 days unless you specify a different number of retries. In its current state, there's a possibility that we would be unable to get the veteran's ICN from the record were its PII to be removed (which can happen in as little as 7 days after submission). 

We should look for & gracefully exit from the job if the PII has been removed. I don't think we'd need to alert in that case, but maybe log it?

Reply:
I added [this](https://github.com/department-of-veterans-affairs/vets-api/pull/10418/commits/16736b784d8e8440c5d2db72064c909ea60852f7). Is the Rails logger sufficient, or would we want to send this to Sentry and/or attach the appeal ID to the log?

Reply:
Rails logging is fine and yes, let's add the ID into the log.